script/cucumber

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
```

Comment:
not blocking - what's the difference between script and bin cucumber? Is there a difference? Or is this some vestigial rails 3.X support that gets auto-installed?

Reply:
Vestigial, I think.